api-docs/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would suggest putting this within `docs/` rather than adding a new workspace if possible